rest/data/project.go

```diff
@@ -259,6 +259,37 @@ func (pc *DBProjectConnector) UpdateProjectVars(projectId string, varsModel *res
 	return nil
 }
 
+func (pc *DBProjectConnector) UpdateProjectVarsByValue(toReplace, replacement string, dryRun bool) ([]*model.ProjectVars, error) {
+	catcher := grip.NewBasicCatcher()
+	matchingProjects, err := model.GetVarsByValue(toReplace)
+	if err != nil {
+		catcher.Add(errors.Wrap(err, "failed to fetch projects with matching value"))
```

Comment:
you can use catcher.Wrap() instead of catcher.Add(errors.Wrap())

Reply:
done